db/migrate/20170714144125_add_deb.rb

```diff
@@ -0,0 +1,34 @@
+class AddDeb < ActiveRecord::Migration
```

Comment:
Rails 5.1 support is about to be merged in foreman, and requres some new syntax for migrations, see these examples:  https://github.com/Katello/katello/pull/7094/files  

Can you go ahead and add that here?